Neos.Utility.Pdo/Classes/PdoHelper.php

```diff
@@ -31,10 +37,17 @@ abstract class PdoHelper
      */
     public static function importSql(\PDO $databaseHandle, string $pdoDriver, string $pathAndFilename)
     {
-        $sql = file($pathAndFilename, FILE_IGNORE_NEW_LINES & FILE_SKIP_EMPTY_LINES);
-        // Remove MySQL style key length delimiters (yuck!) if we are not setting up a MySQL db
-        if ($pdoDriver !== 'mysql') {
-            $sql = preg_replace('/"\([0-9]+\)/', '"', $sql);
+        $path = dirname($pathAndFilename);
+        $filename = basename($pathAndFilename);
+        $dbSpecificPathAndFilename = sprintf('%s/%s.%s', $path, $pdoDriver, $filename);
+        if (file_exists($dbSpecificPathAndFilename)) {
+            $sql = file($dbSpecificPathAndFilename, FILE_IGNORE_NEW_LINES & FILE_SKIP_EMPTY_LINES);
+        } else {
+            $sql = file($pathAndFilename, FILE_IGNORE_NEW_LINES & FILE_SKIP_EMPTY_LINES);
+            // Remove MySQL style key length delimiters (yuck!) if we are not setting up a MySQL db
```

Comment:
Isn't this e superfluous by now, given that we have a mysql specific file?

Reply:
**We** have that file, for our specific case. But the `PdoHelper` can be used by anyone for any case. I'm not going to break that now… ;)

Reply:
Oh of course! That makes sense :)

Reply:
I already broke it with #1764 🤣

Reply:
Well, you intend to break it. :)